server/src/shared/genanki/package.ts

```diff
@@ -35,7 +35,7 @@ export class Package {
             const zip = new JSZip();
 
             const data = db.export();
-            const buffer = new Uint8Array(data).buffer;
+            const buffer = new Uint8Array(data);
 
             zip.file("collection.anki2", buffer);
```

Comment:
I was getting an error about how the contents of `collection.anki` could not be read when `buffer = new Uint8Array(data).buffer`, but `new Uint8Array(data)` seems to be working